test/unit/models/UcrBlockSpec.scala

```diff
@@ -31,18 +31,18 @@ class UcrBlockSpec extends WordSpec with MustMatchers {
         val ucrBlock = UcrBlock(ucr = validDucr, ucrType = Ducr.codeValue)
```

Comment:
Why have you changed this test.  UcrBlock hasn't changed

Reply:
Good catch, it was when I tried to remove `ucrPartNo` I renamed fullUcr to ucr and didn't change it back. Thanks

Reply:
but this confirm what we discussed yesterday, that ucr field contains the ducr with part

Reply:
I pushed a fix in separate commit for it